mix.lock

```diff
@@ -1,10 +1,10 @@
-%{"backoff": {:hex, :backoff, "1.1.1"},
-  "connection": {:hex, :connection, "1.0.2"},
-  "coverex": {:hex, :coverex, "1.4.3"},
-  "db_connection": {:hex, :db_connection, "0.2.0"},
-  "decimal": {:hex, :decimal, "1.1.0"},
-  "hackney": {:hex, :hackney, "1.3.2"},
-  "httpoison": {:hex, :httpoison, "0.7.2"},
-  "idna": {:hex, :idna, "1.0.2"},
-  "poison": {:hex, :poison, "1.4.0"},
-  "ssl_verify_hostname": {:hex, :ssl_verify_hostname, "1.0.5"}}
+%{"backoff": {:hex, :backoff, "1.1.1", "9c60418f672549304c565ae789282c597c7cc2a1eca1e6125a38bf2dd3bd9fad", [:rebar3], []},
```

Comment:
I think we can unlock this as no longer use it in db_connection - unless used by mariaex